README.md

```diff
@@ -123,6 +123,13 @@ Before running on target computer, make sure "Microsoft Visual C++
 Redistributable for Visual Studio 2019" is installed (get it
 [here](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads))
 
+### Creating a Windows installer
+
+- Install [Inno Setup](http://www.jrsoftware.org/isinfo.php),
+- `cargo build --release`,
```

Comment:
This is now missing the `--features dist`

Reply:
Hmm, maybe this should just link to the installer readme, so we don't have to double-maintain it? They are already out of sync.